ui/src/controller/track_decider.ts

```diff
@@ -2868,16 +2868,16 @@ class TrackDecider {
     }
 
     this.trackGroupsToAdd.push(result);
-    this.tracksToAdd.push(this.blankSummaryTrack(id));
+    this.tracksToAdd.push(this.blankSummaryTrack(id, name));
```

Comment:
I think group names, in general, are not unique. All of the processes that have the same number of idle threads will have a subgroup of the same name. Many processes will have a "Memory Usage" group etc. So instead of multiple summary tracks having the same name `''` they'll still have the same names, just a different same name 😉 

I don't see a reason not to use the group ID as the name, as it will be unique and the name of a summary track is never shown anywhere in the UI. At least, I don't think it is.